aws-predictions/src/main/java/com/amplifyframework/predictions/aws/adapter/IdentifyResultTransformers.java

```diff
@@ -108,4 +123,50 @@ public static Pose fromRekognitionPose(@NonNull com.amazonaws.services.rekogniti
 
         return amplifyLandmarks;
     }
+
+    /**
+     * Gets all the binary features from AWS Rekognition's face
+     * details and compiles them into a single list.
+     * @param face the Rekognition face detail object
+     * @return the list of Amplify {@link BinaryFeature}
+     */
+    public static List<BinaryFeature> fromFaceDetail(FaceDetail face) {
+        List<BinaryFeature> features = new ArrayList<>();
```

Comment:
Just use:
```
Arrays.astList(
    new Foo<>("asdf"),
    new Foo<>("asdfeee"),
    new Foo<>("sdfgsdfgr")
)
```

Almost as good as [Java 9 `List.of()`](https://docs.oracle.com/javase/9/docs/api/java/util/List.html#of-E...-), or [Kotlin `listOf()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/list-of.html) :-D

Reply:
haha i was thinking about it, and the only reason I decided against it was to reduce indentation depth. Will fix